README.md

```diff
@@ -16,39 +16,31 @@ To be able to fetch github and cgit repositories, you should create several API
 
 ### Download docker images
 
-Get the last version of rovers spark image:
+Get the latest rovers image:
 
 ```bash
-docker pull quay.io/srcd/rovers
+docker pull srcd/rovers
 ```
 
-Also, you will need Postgres and RabbitMQ
+### Start everything using docker-compose
 
-```bash
-docker pull postgres:9.6-alpine
-docker pull rabbitmq:3-management
-```
-
-### Start everything
+Install [docker-compose](https://docs.docker.com/compose/install/).
 
 Start Rabbit and Postgres
 
 ```bash
-docker run -d --hostname postgres --name postgres -e POSTGRES_PASSWORD=testing -p 5432:5432 -e POSTGRES_USER=testing postgres:9.6-alpine
-```
-```bash
-docker run -d --hostname rabbitmq --name rabbitmq -p 8081:15672 -p 5672:5672 rabbitmq:3-management
+docker-compose -d up rovers-postgres rovers-rabbitmq
```

Comment:
I'm used to run  `docker-compose up --no-deps <service>` to avoid running everything else what's not needed.